document-store/src/integrationTest/java/org/hypertrace/core/documentstore/utils/Utils.java

```diff
@@ -112,4 +112,23 @@ public static void assertDocsAndSizeEqual(
     assertEquals(expected, actual);
     assertEquals(expectedSize, actualSize);
   }
+
+  public static void assertDocsAndSizeEqualWithoutOrder(
+      Iterator<Document> documents, String filePath, int expectedSize) throws IOException {
+    String fileContent = readFileFromResource(filePath).orElseThrow();
+    List<Map<String, Object>> expectedDocs = convertJsonToMap(fileContent);
+
+    List<Map<String, Object>> actualDocs = new ArrayList<>();
```

Comment:
Can this be a HashSet to support `contains` in O(1) though this is used only in test?

Reply:
Will do this in the test case clean-up PR.